packages/flask/src/UIDateTimePicker/Calendar/Calendar.tsx

```diff
@@ -51,6 +53,19 @@ const Column = React.memo(function Column({
     onSelect: (column: number, row: number) => void;
 }) {
     const theme = useTheme();
+
+    /**
+     * Safari below 15 version not support aspect-ratio. So we have to provide width/height for day view.
+     * To create a smooth circle, we need to get the font height.
+     * getFontMesurements() may return undefined, so we must also pass the default value.
+     * The size is calculated for two numbers, so this value needs to be multiplied by two.
+     */
+    const fontHeight =
+        getFontMesurements(TypographyVariants.Action)?.upperline ||
+        UIConstant.calendar.dayCellDefaultFontHeight;
+    const daySize = fontHeight * 2;
```

Comment:
```suggestion
    const style = React.useMemo(() => {
        const fontHeight = getFontMesurements(TypographyVariants.Action)?.upperline || UIConstant.calendar.dayCellDefaultFontHeight;
        const daySize = fontHeight * 2;
        return { width: daySize, height: daySize };
    }, []);
```
Btw, why are you multiplying `fontHeight` by 2, isn't it should be just adding padding multiplied by 2 (i.e. `daySize = fontHeight + (padding * 2)`)?

Reply:
@savelichalex yeah you're right better to use padding*2